### PR TITLE
Basic product SEO

### PR DESCRIPTION
**Pull request by jclariviere**

### CODE REVIEW

File: client/src/main/java/com/arcbees/beestore/client/resources/SeoImages.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        images.put(new SimpleEntry<>(Brand.GSSS, ProductType.MUG), image("http://store.arcbees.com/img/arcbees-shirt.png"));
+        images.put(new SimpleEntry<>(Brand.GSSS, ProductType.THERMOS), image("http://store.arcbees.com/img/arcbees-shirt.png"));
+        images.put(new SimpleEntry<>(Brand.GSSS, ProductType.USB_KEY), image("http://store.arcbees.com/img/arcbees-shirt.png"));
+        images.put(new SimpleEntry<>(Brand.GSSS, ProductType.PHONE_CASE), image("http://store.arcbees.com/img/arcbees-shirt.png"));
+
+        return images;
+    }
+
+    private Image image(String url) {
+        return new Image(url, 600, 315, PNG);
```
Comment (Pacane) [inline client/src/main/java/com/arcbees/beestore/client/resources/SeoImages.java]: extract these to constants

File: server/src/main/java/com/arcbees/beestore/server/guice/CrawlerFilter.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    @Override
+    public void doFilter(ServletRequest request, ServletResponse response, FilterChain filterChain)
+            throws IOException, ServletException {
+        HttpServletRequest req = (HttpServletRequest) request;
+        HttpServletResponse resp = (HttpServletResponse) response;
+
+        String queryString = req.getQueryString();
+        if (queryString != null && queryString.contains(ESCAPED_FRAGMENT)) {
```
Comment (Chris-V) [inline server/src/main/java/com/arcbees/beestore/server/guice/CrawlerFilter.java:56]: I would check that the host is under *.arcbees.com to limit what can use this crawler service

File: client/src/main/java/com/arcbees/beestore/client/resources/SeoImages.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    private final Map<SimpleEntry<Brand, ProductType>, Image> images;
+
+    public SeoImages() {
+        images = initializeImages();
+    }
+
+    private Map<SimpleEntry<Brand, ProductType>, Image> initializeImages() {
+        Map<SimpleEntry<Brand, ProductType>, Image> images = new HashMap<>();
+
+        images.put(new SimpleEntry<>(Brand.ARCBEES, ProductType.SHIRT), image("http://store.arcbees.com/img/arcbees-shirt.png"));
```
Comment (Chris-V) [inline client/src/main/java/com/arcbees/beestore/client/resources/SeoImages.java]: `image()` could prepend `http://store.arcbees.com/img/` to the argument

**Comment by Chris-V** [comment]
"Arcbees LOL" in the image?

**Comment by jclariviere** [comment]
Hmm we wanted some text to see how it looked, the image will be replaced in another PR

**Pull request merged**